source/mechanism.md

```diff
@@ -115,14 +115,46 @@ The primary aim with this deployment was to provide the [dxDAO](https://dutchx.r
 
 ### Some more details to the auctioneer powers:
 #### De- & whitelisting of tokens that generate Magnolia
-TODO
+Whitelisted tokens - if traded in a whitelisted pair - generate Magnolia, which can be used to reduce liquditiy contribution.
+Read above about [Magnolia](https://dutchx.readthedocs.io/en/latest/mechanism.html#magnolia), [Whitelist](https://dutchx.readthedocs.io/en/latest/mechanism.html#whitelist) and [Liquidity contribution](https://dutchx.readthedocs.io/en/latest/mechanism.html#liquidity-contribution).<br/>
+- Whitelisting (and de-whitelisting) can happen by token individually or by entire lists
+- It is not necessary that the token is either listed for trading or actively traded
+- The auctioneer calls the function *updateApprovalOfToken* in the [Token Whitelist contract](https://github.com/gnosis/dx-contracts/blob/master/contracts/base/TokenWhitelist.sol#L28)
+- There is no time lag to the execution of this function
+- It applies immediately and is effective upon users *claiming* their tokens of whitelisted tokens (if traded in a whitelisted pair), also if the auction had finished prior to triggering this function
+
 #### Changing the threshold to start auctions
-TODO
+Once this defined sellVolume (deposit) threshold is reached by both opposite auctions (each), and the prior auctions have finished, the new auctions will commence.
```

Comment:
There's also the case when only one side has reached the threshold and 24hours have passed
https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchange.sol#L799
Not sure if this case is significant enough to merit mentioning